common/apiClient.ts

```diff
@@ -0,0 +1,38 @@
+import { User } from './types'
+import Axios from 'axios'
+
+const host = 'https://application-portal.vercel.app'
+export function getAdminById(id: number) {
+  return Axios.get<User>(`/v1/admin/${id}`, { baseURL: host })
```

Comment:
I think the routes need to be prefixed with `/api`, so this should become `/api/v1/admin/${id}`

Reply:
ya I missed that one- just updated!